instrumentation-api-semconv/src/test/java/io/opentelemetry/instrumentation/api/instrumenter/http/HttpClientExperimentalMetricsTest.java

```diff
@@ -35,24 +36,24 @@ void collectsMetrics() {
 
     Attributes requestAttributes =
         Attributes.builder()
-            .put("http.method", "GET")
-            .put("http.url", "https://localhost:1234/")
-            .put("http.target", "/")
-            .put("http.scheme", "https")
-            .put("net.peer.name", "localhost")
-            .put("net.peer.port", 1234)
-            .put("http.request_content_length", 100)
```

Comment:
Should we include it as `http.request.body.size` ?

Reply:
Oh, it looks like the request body size is put down there in the response attributes? What am I missing?

Reply:
`http.request.body.size` didn't make it into the stable HTTP semconv

Reply:
Well that would explain it, wouldn't it! 😁 Thanks.